[Bug]: Unable to Install latest Version of synfig-studio

### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

synfig-studio

### Expected/Current Behaviour

Was unable to install `synfig-studio`. I've tried to download file by link from manifest but <https://www.fosshub.com/Synfig.html> page is loaded instead of download the file.

### Steps to Reproduce

```console
PS C:\Users\user> scoop install synfig-studio
WARN Scoop uses 'aria2c' for multi-connection downloads.
WARN Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN To disable this warning, run 'scoop config aria2-warning-enabled false'.
Installing 'synfig-studio' (1.4.4) [64bit] from extras bucket
Starting download with aria2 ...
Download: 04/13 03:08:16 [ERROR] CUID#8 - Download aborted. URI=https://download.fosshub.com/Protected/expiretime=1681394486;badurl=aHR0cHM6Ly93d3cuZm9zc2h1Yi5jb20vU3luZmlnLmh0bWw=/9920b4e7f8002a3eca9a067a5a9df6e85110ca95f1309d74f988e921a74Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://download.fosshub.com/Protected/expiretime=1681394486;badurl=aHR0cHM6Ly93d3cuZm9zc2h1Yi5jb20vU3luZmlnLmh0bWw=/9920b4e7f8002a3eca9a067a5a9df6e85110ca95f1309d74f988e921a7418Download: Download Results:
Download: gid |stat|avg speed |path/URI
Download: ======+====+===========+=======================================================
Download: e425e5|ERR | 0B/s|C:/Users/user/scoop/cache/synfig-studio#1.4.4#https_www.fosshub.com_Synfig.html_SynfigStudio-1.4.4-2022.12.25-win64-b8d62.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://download.fosshub.com/Protected/expiretime=1681394486;badurl=aHR0cHM6Ly93d3cuZm9zc2h1Yi5jb20vU3luZmlnLmh0bWw=/9920b4e7f8002a3eca9a067a5a9df6e85110ca95f1309d74f988e921a7418e3c/5b8d2d9359eee027c3d78918/63ac69c9bbb94a1ce9ec4d54/SynfigStudio-1.4.4-2022.12.25-win64-b8d62.zip
referer=https://www.fosshub.com/Synfig.html/
dir=C:\Users\user\scoop\cache
out=synfig-studio#1.4.4#https_www.fosshub.com_Synfig.html_SynfigStudio-1.4.4-2022.12.25-win64-b8d62.zip

ERROR & 'C:\Users\user\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\user\scoop\cache\synfig-studio.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=11992 --continue --summary-interval=0 --auto-save-interval=1
```


### Possible Solution

There are no `SynfigStudio-1.4.4-2022.12.25-win64-b8d62.zip` file on `fosshub.com`. Correct file name is `SynfigStudio-1.4.4-2023.03.30-win64-b8d62.zip`.

### Scoop and Buckets Version

```console
PS> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
7873ba08f (HEAD -> master, origin/master, origin/HEAD) sysinternals@2023.03.30: Fix hash (Closes #11030)

'games' bucket:
0414fca5 (HEAD -> master, origin/master, origin/HEAD) snes9x-dev: Update to version 1.62.3-1266

'main' bucket:
706ea7507 (HEAD -> master, origin/master, origin/HEAD) vector: Update to version 0.29.0

'nerd-fonts' bucket:
d7b401e (HEAD -> master, origin/master, origin/HEAD) SarasaGothic: Update to version 0.40.5

'nonportable' bucket:
26bd563 (HEAD -> master, origin/master, origin/HEAD) spotx-np: Update to version 1.7-20230412-3f430b

'sysinternals' bucket:
1dc6b2f (HEAD -> main, origin/main, origin/HEAD) tcpview: Update to version 4.19

'versions' bucket:
3a226c759 (HEAD -> master, origin/master, origin/HEAD) edgedriver-canary: Add version 114.0.1782.0 (#1143)
```


### Scoop Config

```console
PS> scoop config


last_update       : 2023-04-13T03:06:49.4432863+03:00
use_external_7zip : True
aria2-enabled     : True
scoop_branch      : master
scoop_repo        : https://github.com/ScoopInstaller/Scoop
```


### PowerShell Version

```console
PS> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.19041.2673
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.2673
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```


### Additional Softwares

_No response_

--- a/bucket/synfig-studio.json
+++ b/bucket/synfig-studio.json
@@ -5,12 +5,12 @@
     "license": "GPL-3.0-or-later",
     "architecture": {
         "64bit": {
-            "url": "https://www.fosshub.com/Synfig.html/SynfigStudio-1.4.4-2022.12.25-win64-b8d62.zip",
-            "hash": "d1aa00d079dc679a149db69438688cb8e1414d7369fcbac447a7b805b2dccb41"
+            "url": "https://www.fosshub.com/Synfig.html/SynfigStudio-1.4.4-2023.03.30-win64-b8d62.zip",
+            "hash": "c659cf03c646891abb160e9f0aed7f9cba9038d0d93aff2cb4ef281085055ad8"
         },
         "32bit": {
-            "url": "https://www.fosshub.com/Synfig.html/SynfigStudio-1.4.4-2022.12.25-win32-b8d62.zip",
-            "hash": "4457b5a3db2ebe219d3cc2d53a00683e059742b1c119f28c2ec36ea882256f9e"
+            "url": "https://www.fosshub.com/Synfig.html/SynfigStudio-1.4.4-2023.03.30-win32-b8d62.zip",
+            "hash": "c817ad26b00efc8fa97c2e44824a5740d46ed02f13cd23138e3dc3cff6bd0e5f"
         }
     },
     "bin": "bin\\synfig.exe",

